### ISSUE TITLE
[JENKINS-62419] Indicate the language of the page in the footer

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-21#204]
<p>It would be great to indicate the language of the page being displayed. Even if some localizations are missing, showing a baseline language might be useful.</p>

<p>This locale field should be somehow determined from the response page so that plugins like Locale Plugin are handled correctly (see the filter here: <a href="https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java</a>)</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/oleg_nenashev.png" title="oleg_nenashev's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=oleg_nenashev">oleg_nenashev</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-62419" target="_blank">Indicate the language of the page in the footer</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: accessibility, newbie-friendly, user-experience
<li><b>resolution</b>: Won't Fix
<li><b>resolved</b>: 2025-03-21T13:23:03+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 3
<li><b>imported</b>: 2025-11-22
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>It would be great to indicate the language of the page being displayed. Even if some localizations are missing, showing a baseline language might be useful.</p>

<p>This locale field should be somehow determined from the response page so that plugins like Locale Plugin are handled correctly (see the filter here: <a href="https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/locale-plugin/blob/master/src/main/java/hudson/plugins/locale/LocaleFilter.java</a>)</p></pre>
</details>
</details>
<details><summary><i>4 attachments</i></summary>

- [jenkins-locale-1.png](https://issues.jenkins.io/secure/attachment/51316/jenkins-locale-1.png)
  > ![jenkins-locale-1.png](https://issues.jenkins.io/secure/attachment/51316/jenkins-locale-1.png)
- [jenkins-locale-2.png](https://issues.jenkins.io/secure/attachment/51315/jenkins-locale-2.png)
  > ![jenkins-locale-2.png](https://issues.jenkins.io/secure/attachment/51315/jenkins-locale-2.png)
- [jenkins-locale-3.png](https://issues.jenkins.io/secure/attachment/51314/jenkins-locale-3.png)
  > ![jenkins-locale-3.png](https://issues.jenkins.io/secure/attachment/51314/jenkins-locale-3.png)
- [jenkins-locale-4.png](https://issues.jenkins.io/secure/attachment/51317/jenkins-locale-4.png)
  > ![jenkins-locale-4.png](https://issues.jenkins.io/secure/attachment/51317/jenkins-locale-4.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>sakshi__:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-62419?focusedId=453001&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-453001">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Hi, is this issue resolved? or any new pull request regarding this issue has been made that I can work on?</p>
</pre>
</details>

</ul>
</details>

<p>Hi, is this issue resolved? or any new pull request regarding this issue has been made that I can work on?</p>
